.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on: push
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-10.14, windows-2019, ubuntu-18.04]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
+
+      - name: Install Node.js, NPM and Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: 13
+
+      - name: yarn install
+        run: |
+          yarn install
+
+      - name: yarn test
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          yarn package-ci
```

Comment:
are packaged artifacts only generated with the merge or should we see them listed under the artifact section on the testing page for github actions?

Reply:
It looks like this CI setup doesn't attach artifacts to actions, but to Github releases. Once CI completes next, we should see app installers automatically attached to this draft release https://github.com/makebrainwaves/BrainWaves/releases/tag/v0.14.0